### ISSUE TITLE
Where is the C# Unity SpeechBotConnector sample for Windows or Android?

### ISSUE THREAD

**Issue by aprilspeight** [Azure-Samples/cognitive-services-speech-sdk#730]
I've downloaded the .ZIP file and cannot locate this sample. Could someone please advise as to where it is in the folder? Thank you.

**Comment by aprilspeight** [comment]
The Unity project that's tied to the quickstart does not have a SpeechBotConnectorSample scene nor script. Instead, there is a SampleDialogServiceConnector scene. The script that's in the project does not mention a place to insert the bot ID. I believe the respective Unity project was not attached to this sample.

**Comment by aprilspeight** [comment]
Hi - it's me again :) I've done a bit more snooping around. 

Using the SampleDialogServiceConnector scene, I opened the DialogServiceConnectorSample script that's included in the project. This script _can_ work using a bot created with the bot framework. _However_ the script itself needs to be modified. These two teeny tiny modifications.

1. On line 22, add `public string botID`
2. On line 127, the `botID` previously added needs to be added to the config. Therefore, the line should look like this: `dialogServiceConfig = BotFrameworkConfig.FromSubscription(subscriptionKey, region, botID);`

Back over in the README.md (https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/csharp/unity/VirtualAssistantPreview/), the scene name should be changed to SampleDialogServiceConnector. The script we reference needs to be changed to DialogServiceConnectorSample.cs. We would also need to add in 'Replace `Bot ID` with the bot Microsoft App ID' in the bullet point that talks about the `Canvas` GameObject.

Now, the only thing about the Microsoft App ID is that I'm unsure where to find it in Azure portal. I found mine for my QnA bot by downloading the bot source code from the Azure portal and opening the appsettings.json file. It's the first line of the file. I'll keep looking around in the portal though to see where the ID is. I can confirm it's not on the Overview page as that page only provides the Subscription ID.

In any case, I hope that helps!
